lib/core/src/Cardano/Pool/DB/Sqlite.hs

```diff
@@ -275,9 +288,20 @@ newDBLayer trace fp = do
 
             rights . fmap safeCast <$> rawSql query []
 
+        , putFetchAttempt = \(url, hash) -> do
+            -- NOTE
+            -- assuming SQLite has the same notion of "now" that the host system.
```

Comment:
I'm hoping they can't use different timezones, right?

Reply:
Well, since they're both (the database and the server) on the same host system, I'd assume they both have the same clock.